src/en/news/blog/2022/s3select-intro/index.md

```diff
@@ -62,26 +62,26 @@ Adding the capability to read and process different and common data sources such
 makes the s3select a powerful tool for machine learning.
 
 ## Why s3 storage? 
-S3-storage is reliable, efficient, cheap, and already contains [trillions of objects](https://www.zdnet.com/article/aws-s3-storage-now-holds-over-100-trillion-objects/), It contains many CSV, JSON, and Parquet objects, and these objects contain a huge amount of data to analyze.
+S3-storage is reliable, efficient, cheap, and already contains [trillions of objects](https://www.zdnet.com/article/aws-s3-storage-now-holds-over-100-trillion-objects/). It contains many CSV, JSON, and Parquet objects, and these objects contain a huge amount of data to analyze.
 
-An ETL may convert these objects into efficient format such as Parquet and later run queries on these converted objects.
+An ETL may convert these objects into an efficient format such as Parquet and later run queries on these converted objects.
 
 But it comes with an expensive price, getting all of these objects.
 
 S3select engine that resides on s3-storage can do these jobs for many use cases, saving time and resources.
 
-These semi-structured data reside on S3 storage as “cold” data and as mentioned in huge numbers, with S3select this “locked” data can be accessed efficiently. 
+These semi-structured data reside on S3 storage as “cold” data and as mentioned in huge numbers, with S3select, this “locked” data can be accessed efficiently. 
 
 ## Parquet vs CSV and JSON
-Upon processing CSV or JSON objects, the whole object must be scanned, there is no way to fetch only the relevant columns and rows.
+Upon processing CSV or JSON objects, the whole object must be scanned. There is no way to fetch only the relevant columns and rows.
 
-With the Parquet object, it's different, the engine analyzes the query, and fetches only the relevant columns, using the s3-range-request and the apache-parquet-reader.
+With the Parquet object, it's different. The engine analyzes the query, and fetches only the relevant columns, using the s3-range-request and the apache-parquet-reader.
 
 In that way, it reduces the IOPS on the server side.
 
 And finally … 
 
-S3select Repo is growing, we keep adding new features, and planning new ones.
+S3select Repo is growing, and we keep adding new features, and planning new ones.
```

Comment:
I think the subject should be "S3select" rather than the s3select repo, since the repo isn't growing, the feature is.  It's maybe a bit confusing to suggest that s3select per se is going to grow *a lot*--but computational storage and analytics integration *are*?